server/server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const port = 3000;
```

Comment:
Please save port 3000 for proxy server. Can you choose another port for your app?

Reply:
Ooh, that's right. I always just set 3000 by default, I'll switch to another.